.github/workflows/main.yml

```
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
```

```suggestion
name: Auto Format Documentation
```
Also should update workflow name?

---

^ like the file name

---

renamed the file as well.